docs/changelog.rst

```diff
@@ -1,6 +1,12 @@
 Changelog
 ~~~~~~~~~
 
+0.26.0 (unreleased)
+----------------------
```

Comment:
```suggestion
-------------------

Reply:
Line only as long as the text above

Reply:
Sorry, still haven't gotten used to .rst